prog/weaveproxy/main.go

```diff
@@ -15,6 +15,23 @@ var (
 	defaultListenAddrs = []string{"tcp://0.0.0.0:12375", "unix:///var/run/weave.sock"}
 )
 
+type listOpts struct {
+	values *[]string
```

Comment:
Because it's not in `pkg/`, so it can have dependencies on other bits of Docker, which I don't want to drag in.